kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/TopicsService.java

```diff
@@ -61,6 +64,10 @@
   private final ClusterMapper clusterMapper;
   private final DeserializationService deserializationService;
   private final MetricsCache metricsCache;
+  @org.springframework.beans.factory.annotation.Value("${topic.recreate.maxAttempt:3}")
+  private int maxAttemptToRecreateTopic;
+  @org.springframework.beans.factory.annotation.Value("${topic.recreate.timeout.seconds:5}")
```

Comment:
this settings should be documented, especially considering the existence of default obscure values

Reply:
I've updated readme file with these new values.